sites/all/modules/publizon/includes/publizon.availability.inc

```diff
@@ -79,13 +109,15 @@ function publizon_availability_items(array $local_ids) {
     $key = $to_check[$isbn];
     $results[$key] = $data;
 
-    // Store in cache
-    $static_cache[$key] = $data;
+    if ($use_cache) {
+      // Store in cache.
+      $static_cache[$key] = $data;
```

Comment:
Why not have the static cache out-side the $use_cache. That would be better as this is called more than once on some pages and the static cache is stille checked and is valid for the current user.

Reply:
Fixed!